### DISCUSSION TITLE
Show image from cliphist?

### DISCUSSION THREAD

**Discussion by azzamsa** [anyrun-org/anyrun#129] [Q&A]
Hi, 👋.

How to show image from [cliphist](https://github.com/sentriz/cliphist)?

```
cliphist list | anyrun --plugins libstdin.so | cliphist decode | wl-copy
```

Thank you for anyrun! 🍦

**Comment by noahfraiture** [comment]
Hey, I'm late but is it possible tho ?

**Reply by abenz1267** [reply]
no. either use rofi with the cliphist image plugin (https://github.com/sentriz/cliphist/blob/master/contrib/cliphist-rofi-img) or walker (https://github.com/abenz1267/walker).

**Reply by fixerer** [reply]
Maybe this is a bit of a hack to use thumbnails as icons; but see example from https://github.com/fixerer/anyrun-json#

<img width="575" height="291" alt="image" src="https://github.com/user-attachments/assets/869453fb-8b1f-4761-a624-77015a59ec6c" />
